src/include/migraphx/op/nonmaxsuppression.hpp

```diff
@@ -140,6 +141,7 @@ struct nonmaxsuppression
     {
         std::array<double, 2> x;
         std::array<double, 2> y;
+        double stored_area;
```

Comment:
Area is pretty trivial to compute so I dont think we need to cache this. This could also lead to errors with reading uninitialized value as well.

Reply:
Done